qmetaobject/src/qtquickcontrols2.rs

```diff
@@ -1,9 +1,83 @@
-/// Refer to the Qt documentation for QQuickStyle
-pub struct QQuickStyle {}
+/* Copyright (C) 2019 Tim Süberkrüb <dev@timsueberkrueb.io>
+   Copyright (C) 2020 Jonah Brüchert <jbb@kaidan.im>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+associated documentation files (the "Software"), to deal in the Software without restriction,
+including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so,
+subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial
+portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+use crate::qttypes::QString;
+
+cpp! {{
+    #include <QtQuickControls2/QQuickStyle>
+}}
+
+cpp_class!(
+/// Wrapper around Qt's QQuickStyle class
+    pub unsafe struct QQuickStyle as "QQuickStyle"
```

Comment:
Everything is static within QQuickStyle, so there is no need to wrap it with cpp_class.  (cpp_class will add the Default, Drop, Clone impl, but we don't need it as we don't need an instance of QQuickStyle)